doc/source/examples/extended_examples/sfem/sfem.py

```diff
@@ -0,0 +1,758 @@
+        for element_id in element_ids:
+            material_property += 1
+            mapdl.get("ELEMENT_ID", "ELEM", element_id, "CENT", "X")
+            element_centroid_x_coord = mapdl.parameters["ELEMENT_ID"]
+            mapdl.esel(
+                "S", "CENT", "X", element_centroid_x_coord
+            )  # Select all elements having this coordinate as centroid
+
+            # Evaluate young's modulus at this material point
+            young_modulus_value = young_modulus_realization(
+                cosine_frequency_list,
+                cosine_eigen_values,
+                cosine_constants,
+                cosine_random_variables_set,
+                sine_frequency_list,
+                sine_eigen_values,
+                sine_constants,
+                sine_random_variables_set,
+                domain,
+                element_centroid_x_coord,
+            )
+
+            mapdl.mp(
+                "EX", f"{material_property}", young_modulus_value
+            )  # Define property ID, assign young's modulus
+            mapdl.mp(
+                "NUXY", f"{material_property}", poisson_ratio
+            )  # Assign poisson ratio
+            mapdl.mpchg(
+                material_property, "ALL"
+            )  # Assign property to selected elements
+
+        mapdl.allsel()
+
+        mapdl.d("FIXED_END", lab="UX", value=0, lab2="UY")  # Apply fixed end BC
+        mapdl.f("LOAD_APPLICATION_POINT", lab="FY", value=load)  # Apply load BC
+        mapdl.solve()
+
+        # Displacement probe point - where Uy results will be extracted
+        displacement_probe_point = mapdl.queries.node(length, 0, 0)
+        displacement = mapdl.get_value("NODE", displacement_probe_point, "U", "Y")
+
+        simulation_results[simulation] = displacement
+
+        mapdl.mpdele("ALL", "ALL")
+        if (simulation + 1) % 10 == 0:
+            print(f"Completed {simulation + 1} simulations ...")
+
+    mapdl.exit()
+    print()
+    print("All simulations completed!")
+
+    return simulation_results
+
+
+# Run the simulations
+import time
+
+start = time.time()
+simulation_results = run_simulations(4, 1, 0.2, 0.1, 5000)
+end = time.time()
+print(
+    "Simulation took {} min {} s".format(
+        int((end - start) // 60), int((end - start) % 60)
+    )
+)
+
+# Perform statistical post processing and plot the pdf
+import scipy.stats as stats
+
+kde = stats.gaussian_kde(simulation_results)  # Kernel density estimate
+
+fig, ax = plt.subplots()
+fig.set_size_inches(15, 8)
+x_eval = np.linspace(min(simulation_results), max(simulation_results), num=200)
+ax.plot(x_eval, kde.pdf(x_eval), "r-", label="PDF of response $u$")
+plt.xlabel("Displacement in (m)")
+ax.legend()
+plt.show()
+
+# We can then proceed to evaluate the probability that response u is less than 0.2 m
+probability = kde.integrate_box_1d(-0.2, x_eval[-1])
+print(f"The probability that u is less than 0.2 m is {probability:.0%}")
```

Comment:
```suggestion
print(f"The probability that u is less than 0.2 m is {probability:.0%}.")
```
Messages should also have concluding periods.